docs/v.list

```diff
@@ -18,15 +18,15 @@
 
     <var name="coroutinesEapVersion" value="1.6.0-RC3" type="string"/>
 
-    <var name="minGradleVersion" value="6.7.1" type="string"/>
+    <var name="minGradleVersion" value="6.7.3" type="string"/>
```

Comment:
If these Gradle and AGP versions are applicable only to the Beta, we shouldn't change them.
These versions specified only for stable release (no `Eap` in the name of variable).